scripts/api-extractor-for-branch.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
Looking ahead since the release branching strategy is still in discussion, would it be possible to extract the merge target from the PR info, and not always use develop?  If the PR merge destination is something else than develop, the diff will be misleading/inaccurate.

Reply:
I think it does that or something is not working correctly. `develop` is set just as a fallback branch. Bot script passes PR branch as an argument to this script and it is used (look line 5 of this file)